File: TicketReservation/src/main/java/pl/ticketreservation/ticket/TicketDTO.java
package pl.ticketreservation.ticket;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import pl.ticketreservation.screening.Screening;
import pl.ticketreservation.seat.Seat;
import pl.ticketreservation.user.User;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class TicketDTO {
    private Ticket ticket;
    private Screening screening;
    private User user;
    private Seat seat;
}

File: ticket-reservation-app/src/app/cinema/models/ticket-dto.ts
import {Ticket} from "./ticket";
import {Screening} from "./screening";
import {User} from "./user";
import {Seat} from "./seat";
import {Movie} from "./movie";

export class TicketDto {
  ticket: Ticket = new Ticket()
  movie: Movie = new Movie()
  screening: Screening = new Screening()
  user: User = new User()
  seat: Seat = new Seat()

}
